stable/rocketchat/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: rocketchat
-version: 1.1.15
-appVersion: 1.3.1
+version: 1.1.16
```

Comment:
Maybe a major version bump is in order to reflect the RocketChat major version bump?